environment-py3.yml

```
@@ -83,6 +83,8 @@ dependencies:
   - typing >=3.6.6
   - pyyaml
   - pip:
+    - git+https://github.com/executablebooks/sphinx-panels.git@v0.6.0
```

Why do these packages need to be installed from their git repos, and not from standard locations (pypi has both of them).
And why these specific versions?

---

> Why do these packages need to be installed from their git repos, and not from standard locations (pypi has both of them).

Apparently, this theme suits us and this PR can be considered for merging. Then if I understand correctly, these packages should be added to `DIRACOS2`.

> And why these specific versions?

I don't think we need to stick to versions of these packages. I just take the latest versions of the packages.

---

```suggestion
    - sphinx-panels
```

Should be enough?  (or `>=0.6.0` if needed) Similar for sphinx_rtd_theme
(can this be in dependencies? @chrisburr ?)
As this environment-py3 file is used in readthedocs and the CI the packages also need to be here, but with as few version contraints as possible.